R/rdeephaven/lib/cpp-dependencies/build-dependencies.sh

```diff
@@ -0,0 +1,390 @@
+#!/bin/bash
```

Comment:
Is this file somewhere else in the repo, or is this the only location?  If somewhere else, this should be a symlink.

Reply:
There is a modification to this version of it that builds Cares as a static library, because that is what R wants. So, it's not identical to [this](https://github.com/deephaven/deephaven-base-images/blob/main/cpp-client/build-dependencies.sh), which also doesn't live in the same repo.